### ISSUE TITLE
Latest system-upgrade-controller has no system-upgrade-controller.yaml

### ISSUE THREAD

**Issue by clouedoc** [vitobotta/hetzner-k3s#237]
I get this error while trying to add a new Node

```
Deploying k3s System Upgrade Controller...
error: unable to read URL "https://github.com/rancher/system-upgrade-controller/releases/latest/download/system-upgrade-controller.yaml", server reported 404 Not Found, status code=404
Failed to deploy k3s System Upgrade Controller:
#<Util::Shell::Result:0x10ac1efa0>
```

**Comment by clouedoc** [comment]
for info: hot-patching the binary didn't work:

[1]    26700 killed     hetzner-k3s

```bash
LC_ALL=C sed -i '' 's|system-upgrade-controller/releases/latest|system-upgrade-controller/releases/v0.11.0|g' /opt/homebrew/Cellar/hetzner_k3s/1.1.2/bin/hetzner-k3s
```

**Comment by vitobotta** [comment]
I have just released 1.1.3 which fixes the URL for the current version and also allows you to configure the URLs for the various manifests. Please give this new release a try and let me know if you run into any issues with this change.

**Comment by clouedoc** [comment]
Thanks for fixing this!
I did not try your release since I fixed my issue in the meantime; but I'm now switching to 1.1.3 and I'll report if it arises again.
(I just launched a `create` to see what happens 👀)

---

For your information: the latest Homebrew version is still 1.1.2 instead of 1.1.3.
I had to follow the manual installation instructions. (not sure if that's super relevant, but I thought that it might help 🙂)

---

_For future issue readers_: I have handled my issue by cloning the repository, installing Crystal+dependencies, editing the URL in the source code, and aliasing the long crystal run command to `hetzner-k3s`
